core/liquidity/v2/amendments.go

```diff
@@ -16,23 +16,21 @@ import (
 	"context"
 	"errors"
 
+	"code.vegaprotocol.io/vega/core/events"
 	"code.vegaprotocol.io/vega/core/types"
 )
 
 var ErrPartyHaveNoLiquidityProvision = errors.New("party have no liquidity provision")
 
-func (e *Engine) AmendLiquidityProvision(
-	ctx context.Context,
-	lpa *types.LiquidityProvisionAmendment,
-	party string,
-) error {
+func (e *Engine) AmendLiquidityProvision(ctx context.Context, lpa *types.LiquidityProvisionAmendment, party string) error {
 	if err := e.CanAmend(lpa, party); err != nil {
 		return err
 	}
 
 	// LP exists, checked in the previous func.
 	lp, _ := e.provisions.Get(party)
 	updatedLp := e.createAmendedProvision(lp, lpa)
+	e.broker.Send(events.NewLiquidityProvisionEvent(ctx, updatedLp))
```

Comment:
well, it should be triggered with pending for sure, also I'll change it a bit because it may be applied immediately. I'll also fix submit because in there it's emitted with active immediately rather than pending